python/perfetto/trace_processor/api.py

```diff
@@ -166,6 +166,30 @@ def metric(self, metrics: List[str]):
     metrics.ParseFromString(response.metrics)
     return metrics
 
+  def trace_summary(self,
+                    metric_ids: List[str],
+                    specs: List[str],
+                    metadata_query_id: str | None = None):
+    """Returns the trace summary data corresponding to the passed in metric
+    IDs and specs. Raises TraceProcessorException if the response returns with
+    an error.
+
+    Args:
+      metric_ids: A list of metric IDs as defined in TraceMetrics
+      specs: A list of textproto specs to be used for the summary
```

Comment:
Why restrict to just texproto specs? Should we not support List[bytes] for binary specs?